File: Week_2/problem_10-Lowest_number.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/219774/problem/E

#include <iostream>
#include <vector>
#include <climits>  // For INT_MAX
using namespace std;

int main()
{
    int n, lowest_num = INT_MAX, lowest_num_index;
    cin >> n;
    vector<int> ar(n);

    for (int i = 0; i < n; i++)
    {
        cin >> ar[i];

        if (ar[i] < lowest_num)
        {
            lowest_num = ar[i];
            lowest_num_index = i + 1;  // 1-based index
        }
    }

    cout << lowest_num << " " << lowest_num_index << endl;

    return 0;
}

File: Week_2/problem_10-Lowest_number.py
# https://codeforces.com/group/MWSDmqGsZm/contest/219774/problem/E

n = int(input())
ar = list(map(int, input().split()))

lowest_num = float('inf')  # Equivalent to INT_MAX in C++
lowest_num_index = -1

for i in range(n):
    if ar[i] < lowest_num:
        lowest_num = ar[i]
        lowest_num_index = i + 1  # 1-based index

print(lowest_num, lowest_num_index)
